rosetta-tools/src/main/java/com/regnosys/rosetta/tools/modelimport/XsdTypeImport.java

```diff
@@ -593,4 +596,26 @@ public Attribute createValueAttribute(ImportTargetConfig config) {
     private boolean isMulti(String maxOccurs) {
         return maxOccurs.equals(UNBOUNDED) || Integer.parseInt(maxOccurs) > 1;
     }
+
+	private Attribute createPlaceholderForAny(XsdAny any, ImportTargetConfig config, ChoiceGroup choiceGroup, RosettaXsdMapping xsdMapping) {
+		StringBuilder docs = new StringBuilder("Placeholder for xsd:any: ")
+				.append("Min Occurs: ").append(any.getMinOccurs())
+				.append("; Max Occurs: ").append(any.getMaxOccurs())
+				.append("; Namespace: ").append(any.getNamespace())
+				.append("; Process Contents: ").append(any.getProcessContents())
+				.append(".");
+
+		Attribute attribute = createAttribute(
+				"anyPlaceholder",
```

Comment:
Minor comment: perhaps put this name in a static final variable - just to avoid magic strings.